patroni/ha.py

```diff
@@ -326,20 +326,20 @@ def get_effective_tags(self) -> Dict[str, Any]:
             tags['nosync'] = True
         return tags
 
-    def notify_citus_coordinator(self, event: str) -> None:
-        if self.state_handler.citus_handler.is_worker():
-            coordinator = self.dcs.get_citus_coordinator()
+    def notify_mpp_coordinator(self, event: str) -> None:
+        if self.state_handler.mpp_handler.is_worker():
+            coordinator = self.dcs.get_mpp_coordinator()
             if coordinator and coordinator.leader and coordinator.leader.conn_url:
                 try:
                     data = {'type': event,
-                            'group': self.state_handler.citus_handler.group,
+                            'group': self.state_handler.mpp_handler.group,
                             'leader': self.state_handler.name,
                             'timeout': self.dcs.ttl,
                             'cooldown': self.patroni.config['retry_timeout']}
                     timeout = self.dcs.ttl if event == 'before_demote' else 2
-                    self.patroni.request(coordinator.leader.member, 'post', 'citus', data, timeout=timeout, retries=0)
+                    self.patroni.request(coordinator.leader.member, 'post', 'mpp', data, timeout=timeout, retries=0)
```

Comment:
Yeah, that's a problem. Should we leave this as 'citus' and leave a comment describing this for now, and change this later when major version changes?

Reply:
There are two options:
1. Check Patroni version from the `leader.member`, and make a decision based on it.
2. Make a decision based on mpp_handler class name, and call /citus endpoint in case of Citus.

Reply:
Thanks for the advice, I kind of prefer the first option, since finally we would get rid of /citus endpoint. So I will implement option 1 for now and wait for more comments.

Reply:
> finally we would get rid of /citus endpoint

We will not. Keeping it costs nothing, but make code backward compatible.

Reply:
> > finally we would get rid of /citus endpoint
> 
> We will not. Keeping it costs nothing, but make code backward compatible.

Make sense, then I guess option 2 should be more reasonable, citus use /citus endpoint,
all other mpp clusters use /mpp.